packages/www/pages/docs/guides/debugging-guide.mdx

```diff
@@ -0,0 +1,71 @@
+
+## Step 2
+
+Does anything playback in *your* own player?
+
+- **No:** Go to the next step.
+- **Yes, but the quality is poor:** Go to step 5.
+
+## Step 3
+
+Does anything playback in the [Livepeer.com test players](/app/test-player)?
+
+- **Yes, the quality is great:** There may be something wrong with your player configuration.
+- **No:** Go to the next step.
+- **Yes, but the quality is poor:** Go to step 5
+
+## Step 4
+
+Check the stream status below the Livepeer.com test player. Is it active?
+
+- **No:** No video is being ingested. Confirm your or your streamer’s broadcast software RTMP configuration is set up properly. 
+- **Yes:** Go to the next step.
+
+## Step 5
+
+Check if you are transcoding. Is there only one `profile` or zero `profiles` listed next to `Playback settings` below the Livepeer.com test player? 
+
+- **No:** You are not transcoding your source stream, and only the source profile is available for playback (no ABR) via the Livepeer.com CDN. There may be something wrong with your source stream. Confirm the keyframe interval is set to 2 in your broadcasting software. Try switching the broadcast software internet provider.
+<br />Or, depending on the resolution, fps and bitrate of your source and the bandwidth of your internet, playback may be slow and choppy. Consider transcoding your ingested video to take advantage of ABR in playback.
+- **Yes:** If two or more `profiles` are listed. Go to the next step.
+
+## Step 6
+
+Do all the `profiles` have the same FPS? Between 5 FPS and 60 FPS work best. Setting FPS to `0` is also acceptable and passes through the source FPS to the transcoded renditions.
+
+- **No:** Your `profiles` FPS is configured incorrectly. Set all `profiles` FPS to the same value. 
+- **Yes:** Go to the next step.
+
+## Step 7
+
+Check if something is wrong with the source stream. 
+
+The left player is configured to only playback the highest resolution available. Except for in rare occassions, the highest resolution is non-transcoded source stream, as served by our CDN. 
+
+Compare the source in the left player with the transcoded renditions in the player on the right (ABR). You can manually switch between transcoded renditions by clicking on the &#8942; symbol in the lower right corner of the player. Do the renditions on the right player show more problems than in the left source?
+
+- **No:** If the problems in the source-only stream are also present in the transcoded renditions, there is something wrong with your source stream. Confirm the keyframe interval is set to 2 in your broadcasting software. Try switching the broadcast software internet provider.
+- **Yes:** If you created your stream via the Livepeer.com API, confirm your `profile bitrate` settings are [reasonable](https://developer.apple.com/documentation/http_live_streaming/hls_authoring_specification_for_apple_devices). And if they are, reach out to Livepeer.com for further help, <a href="mailto:help@livepeer.com">help@livepeer.com</a>.
```

Comment:
How can they confirm that they're "reasonable" through that URL? Perhaps add "take a look at Table 1 in the `Video` Section under `General Authoring Requirements` and compare that to your transcoding settings"

Reply:
Just trying to make sure the guide is as explicit as we can make it, since goal here is to prevent the highest number of questions possible. Don't see any harm to adding extra information. 

If a beginner user is using this guide to debug, they'll reach out to ask the question. That said, I see your point on beginner users being more likely to use livepeer.com and more advanced users using API directly. Perhaps we can specify which steps apply to a livepeer.com user and someone using the API, or separate them. Would make it easier to taylor info to beginner v. advanced. Open to ideas.